scheduler/cups-exec.c

```diff
@@ -154,7 +154,7 @@ main(int  argc,				/* I - Number of command-line args */
   {
     cups_file_t	*fp;			/* File */
     char	line[1024];		/* Line from file */
-    int		linenum = 0;		/* Line number in file */
+    unsigned	linenum = 0;		/* Line number in file */
```

Comment:
OK, didn't notice this change before, but I'm not sure why you are making it?

Reply:
Linenum is never negative.